### ISSUE TITLE
Create "All Sources" tab

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#524]
## Description 

Create the "All Sources" tab and have it permanently selected until we implement other options in the future, see branding bar in Zeplin: https://app.zeplin.io/project/5c807ea562f734bd2756b243/styleguide/components?coid=5ca3e2f29972713e00535626

![all-sources](https://user-images.githubusercontent.com/4522213/62579938-e8de6580-b859-11e9-9071-0150e5ffecc0.png)

**Comment by ninavizz** [comment]
Updated design & scope/purpose, since this ticket was initially created. 

TL;DR, the "All Sources" tab will serve a purpose in filtering messages for users, by giving them a "Show All" view to return to. As such, it has been casually discussed that this control will be useful at that time, but not before. Hence the "Help Wanted" tag being removed, as filtering is not on the near term horizon—however users have commented that they would find it to be useful. 

[Figma file](https://www.figma.com/file/YejZGCIGZikX55LDcfvcYY/Client-%E2%89%A5-Q4-2021?node-id=0%3A8133), [Figma Design System](https://www.figma.com/file/vs52zFnp5zpEHfCnTI0R3U/SDW-Design-System?node-id=51%3A2277) for the Workstation project have also since replaced all Zeplin artifacts.

<img width="256" alt="Screen Shot 2021-11-24 at 2 04 31 PM" src="https://user-images.githubusercontent.com/8262612/143318517-947c0d91-0992-486a-aa8a-80e8f1590282.png">
